bpi_features/bpi_features.features.user_permission.inc

```diff
@@ -1,90 +0,0 @@
-<?php
```

Comment:
Wouldn't they also just be overriden by the permissions setup by the Secure Permissions module?